report-service/src/main/java/gov/cdc/dataingestion/report/integration/service/schema/ElementExtensions.java

```diff
@@ -0,0 +1,103 @@
+package gov.cdc.dataingestion.report.integration.service.schema;
+
+import gov.cdc.dataingestion.report.model.Element;
+import org.checkerframework.common.returnsreceiver.qual.This;
+
+/**
+ * Extension for Elements.
+ */
+public final class ElementExtensions {
+
+    private ElementExtensions() {
+        throw new IllegalStateException("Utility class");
+    }
+
+    /**
+     * Adds missing elements from base element.
+      * @param thiz element to update
+     * @param baseElement base element.
+     * @return Element with merged properties from base.
+     */
+    public static Element mergeWithBase(
+            @This final Element thiz,
+            final Element baseElement) {
+        return new Element(
```

Comment:
There is an API for this, e.g. 

Apache Commons BeanUtils.copyProperties( (Object dest, Object orig)